<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>companyWindow</class>
 <widget class="QDialog" name="companyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firma</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>nazwa firmy:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLineEdit" name="output_name"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>miasto:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>NIP:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="output_city"/>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLineEdit" name="output_NIP"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>ulica:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>REGON:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="output_street"/>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLineEdit" name="output_REGON"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>numer:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>KRS:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLineEdit" name="output_number"/>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="QLineEdit" name="output_KRS"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>kod pocztowy:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>PESEL:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLineEdit" name="output_postalCode"/>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QLineEdit" name="output_PESEL"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>email:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="2">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>telefon:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLineEdit" name="output_email"/>
   </item>
   <item row="11" column="2" colspan="2">
    <widget class="QLineEdit" name="output_phone"/>
   </item>
   <item row="12" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Aktualizuj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Wróć</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>output_name</tabstop>
  <tabstop>output_city</tabstop>
  <tabstop>output_street</tabstop>
  <tabstop>output_number</tabstop>
  <tabstop>output_postalCode</tabstop>
  <tabstop>output_email</tabstop>
  <tabstop>output_NIP</tabstop>
  <tabstop>output_REGON</tabstop>
  <tabstop>output_KRS</tabstop>
  <tabstop>output_PESEL</tabstop>
  <tabstop>output_phone</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
